modules/actors/actor-ffg.js

```diff
@@ -338,12 +338,18 @@ export class ActorFFG extends Actor {
           if (item.type === "armour" && item?.data?.equippable?.equipped) {
             const equippedEncumbrance = +item.data.encumbrance.adjusted - 3;
             encum += equippedEncumbrance > 0 ? equippedEncumbrance : 0;
+          } else if (item.type === "armour" || item.type === "weapon" || item.type === "shipweapon") {
+            let count = 1;
+            if (item.data?.quantity?.value) {
+              count = item.data.quantity.value;
+            }
+            encum += ((item.data?.encumbrance?.adjusted !== undefined) ? item.data?.encumbrance?.adjusted : item.data?.encumbrance?.value) * count;
           } else {
             let count = 1;
             if (item.data?.quantity?.value) {
               count = item.data.quantity.value;
             }
-            encum += (item.data?.encumbrance?.adjusted !== undefined ? item.data?.encumbrance?.adjusted : item.data?.encumbrance?.value) * count;
+            encum += item.data?.encumbrance?.value * count;
```

Comment:
What's the reasoning behind removing the adjusted toggle here?

There might be an argument for make encumbrance.adjusted our default display value on all sheets, then we'll easily spot places it's failing. It should exist on all items now as it's in the template.json so should get saved to the database as part of their 0.8.x schema.

Migrated items from old world are likely getting set to the default template 0 which could be catching us out if we're not setting and then saving it somewhere.

Reply:
The adjusted on anything besides weapons/shipweapons/armor is getting stuck at 0 so it isn't getting calculated. I guess the full fix should be to make sure the adjusted was getting updated.

Reply:
Yeah so that'll be a 0.8.x issue with how and when data is persisted to the backend. Only database objects are sent to things like rolls and chat templates etc, so we need to make sure we're actively using the Document#update API in the right places.

All items should at least have the encumbrance.adjusted element as part of their data schema so we just need to make sure we're saving it to the backend wherever we're setting it.

No more being fast and loose with arbitrary data. 😞 It's one of the main reasons I think the system is due a refactor.. now that Foundry has a more consistent backend we probably need to work in the same way. It's a hangover from how the system was thrown together by me rapidly and without much planning, and then later expanded on by dragkun and others.

Reply:
That said I think this works as a quick fix, as I can't see any places gear items would need their adjusted value used in calc (encumbrance modifiers only effect character encumbrance threshold, not the encumbrance value of the item they're on).